rtos/TARGET_CORTEX/TOOLCHAIN_ARM_STD/mbed_boot_arm_std.c

```diff
@@ -58,23 +58,11 @@ extern uint32_t Image$$RW_IRAM1$$ZI$$Limit[];
  */
 void __rt_entry(void)
 {
-    unsigned char *free_start = HEAP_START;
-    uint32_t free_size = HEAP_SIZE;
-
-#ifdef ISR_STACK_START
-    /* Interrupt stack explicitly specified */
-    mbed_stack_isr_size = ISR_STACK_SIZE;
+    mbed_stack_isr_size = (unsigned char *)ISR_STACK_SIZE;
```

Comment:
If you plan to define `ARM_LIB_HEAP` in all arm linker scripts I suggest to remove above blocks:
`#if !defined(ISR_STACK_SIZE)` and `#if !defined(HEAP_START)` and define
isr stack/heap start/size using linker script symbols directly.

Reply:
I am not planning to update ARM_LIB_HEAP in this PR, addition of that might come in next minor release. Will leave above blocks as is for time being

Reply:
ISR_Stack_Size/Start and related defines removed, all stack information now picked from linker files